.changeset/afraid-coats-laugh.md

```diff
@@ -0,0 +1,5 @@
+---
+"react-magma-dom": major
+---
+
+feat(buttons): adding in color changes throughout applicable components
```

Comment:
We should note here that `outline` and `success` buttons went away because those are breaking changes.